### PR TITLE
New custom hybrid model :)

### PR DESCRIPTION
**Pull request by Scrabouillmax**

### CODE REVIEW

File: dataset_tools.py [hunk 1]
```diff
@@ -179,6 +178,55 @@ def preprocess_batch_hybrid(pixelstring_batch, emotions_batch, DEVICE, with_data
...
+    kp_descriptors = [detector.compute_descriptors(np.array(im)) for im in flipped_imgs]
+    flipped_imgs = [img for i, img in enumerate(flipped_imgs) if kp_descriptors[i][1] is not None]
+    groundtruth = torch.stack(tuple([
+        groundtruth[i, :] for i in range(len(kp_descriptors)) if kp_descriptors[i][1] is not None
+    ]))
+    kp_descriptors = [x for x in kp_descriptors if x[1] is not None]
+
+    descriptor_tensors = []
+    for img_key_points, img_descriptors in kp_descriptors:
+        img_tensor = torch.zeros([130, 48, 48], dtype=torch.float)
```
Comment (riadghorra) [inline dataset_tools.py:212]: du coup tu fais une sorte de 0 padding pour gérer les tailles différentes de descripteurs ?

Comment (Scrabouillmax) [reply dataset_tools.py:212]: oui c'est ca je cree l'"image" associee aux descripteurs: 48*48*130 (128 de descripteurs et 2 pour les infos sur les keypoints). 
Pour tous les pixels où il n'y a pas de descripteurs, je mets des 0. Du coup y a enormement de 0 c'est pour ca que je commence par reduire le volume de 48*48*130 à 24*24*32 dans les premieres couches du classifier
